routes/tasks.js

```diff
@@ -13,6 +13,7 @@ router.get("/self", authenticate, tasks.getSelfTasks);
 router.get("/overdue", authenticate, authorizeRoles([SUPERUSER]), tasks.overdueTasks);
 router.post("/", authenticate, authorizeRoles([APPOWNER, SUPERUSER]), createTask, tasks.addNewTask);
 router.patch("/:id", authenticate, authorizeRoles([APPOWNER, SUPERUSER]), updateTask, tasks.updateTask);
+router.get("/details/:id", tasks.getTask);
```

Comment:
why not just do `/:id` for the task API I believe that would be a better way to manage the API, I do acknowledge that it will have a conflict with `/:username` but why not change that instead? What do you think?
cause ofcourse we will have to edit all the consumers of this `/:username` endpoint but that should work for the betterment in the long run as we would be able to follow a standard way of doing CRUD operations on every collection we have.

Reply:
this will be done in another PR.